### ISSUE TITLE
Add benchmarks to more easily measure performance

### ISSUE THREAD

**Issue by Jacalz** [go-text/render#4]
The measurements in https://github.com/fyne-io/fyne/pull/3633 seem to indicate that the performance is slightly worse with this renderer than the old one. While the memory usage improvements more than make up for that (yay!), we should make sure to have useful benchmarks to allow us to tune the implementation further.

**Comment by andydotxyz** [comment]
Yes indeed. I'd held off comparing to golang.org/x/text due to a bug but it seems there is movement on that so this should be explored.
